### PR TITLE
Update MAJA info.json

### PR DESCRIPTION
**Pull request by moyi4681**

### CODE REVIEW

File: keyboards/kbdfans/maja/info.json [hunk 1]
```diff
@@ -1,80 +1,12 @@
...
-    "keyboard_name": "maja",
-    "url": "",
-    "maintainer": "KBDFANS",
-    "width": 18.75,
-    "height": 5.5,
+    "keyboard_name": "MAJA", 
+    "url": "", 
+    "maintainer": "DZTECH", 
+    "width": 17, 
+    "height": 5,
```
Comment (noroadsleft) [inline keyboards/kbdfans/maja/info.json:6]:
```suggestion
    "keyboard_name": "MAJA",
    "url": "",
    "maintainer": "DZTECH",
    "width": 18.75,
    "height": 5.5,
```

File: keyboards/kbdfans/maja/info.json [hunk 2]
```diff
@@ -1,80 +1,12 @@
...
...l":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":7.75, "y":2}, {"label":"J", "x":8.75, "y":2}, {"label":"K", "x":9.75, "y":2}, {"label":"L", "x":10.75, "y":2}, {"label":":", "x":11.75, "y":2}, {"label":"\"", "x":12.75, "y":2}, {"label":"Enter", "x":13.75, "y":2, "w":2.25}, {"label":"Page Down", "x":16, "y":2}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"x":12.25, "y":3}, {"label":"Shift", "x":13.25, "y":3, "w":1.75}, {"label":"\u2191", "x":15, "y":3}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"x":2.75, "y":4, "w":1.5}, {"label":"Alt", "x":4.25, "y":4, "w":2}, {"x":6.25, "y":4, "w":1.25}, {"label":"Alt", "x":8.75, "y":4, "w":2.75}, {"label":"Fn", "x":11.5, "y":4, "w":1.5}, {"label":"\u2190", "x":14, "y":4}, {"label":"\u2193", "x":15, "y":4}, {"label":"\u2192", "x":16, "y":4}]
```
Comment (noroadsleft) [inline keyboards/kbdfans/maja/info.json:9]:
```suggestion
            "layout": [
                {"x": 0.75, "y": 0.25},
                {"x": 1.75, "y": 0.25},
                {"x": 2.75, "y": 0},
                {"x": 3.75, "y": 0.25},
                {"x": 4.75, "y": 0.25},
                {"x": 5.75, "y": 0.25},
                {"x": 6.75, "y": 0.25},
                {"x": 8.75, "y": 0.25},
                {"x": 9.75, "y": 0.25},
                {"x": 10.75, "y": 0.25},
                {"x": 11.75, "y": 0.25},
                {"x": 12.75, "y": 0},
                {"x": 13.75, "y": 0.25},
                {"x": 14.75, "y": 0.25, "w": 2},
                {"x": 17.75, "y": 0.25},
                {"x": 0.5, "y": 1.25, "w": 1.5},
                {"x": 2, "y": 1.25},
                {"x": 3, "y": 1.25},
                {"x": 4, "y": 1.25},
                {"x": 5, "y": 1.25},
                {"x": 6, "y": 1.25},
                {"x": 8.5, "y": 1.25},
                {"x": 9.5, "y": 1.25},
                {"x": 10.5, "y": 1.25},
                {"x": 11.5, "y": 1.25},
                {"x": 12.5, "y": 1.25},
                {"x": 13.5, "y": 1.25},
                {"x": 14.5, "y": 1.25},
                {"x": 15.5, "y": 1.25, "w": 1.5},
                {"x": 17.75, "y": 1.25},
                {"x": 0.25, "y": 2.25, "w": 1.75},
                {"x": 2, "y": 2.25},
                {"x": 3, "y": 2.25},
                {"x": 4, "y": 2.25},
                {"x": 5, "y": 2.25},
                {"x": 6, "y": 2.25},
                {"x": 9, "y": 2.25},
                {"x": 10, "y": 2.25},
                {"x": 11, "y": 2.25},
                {"x": 12, "y": 2.25},
                {"x": 13, "y": 2.25},
                {"x": 14, "y": 2.25},
                {"x": 15, "y": 2.25, "w": 2.25},
                {"x": 17.75, "y": 2.25},
                {"x": 0, "y": 3.25, "w": 2.25},
                {"x": 2.25, "y": 3.25},
                {"x": 3.25, "y": 3.25},
                {"x": 4.25, "y": 3.25},
                {"x": 5.25, "y": 3.25},
                {"x": 6.25, "y": 3.25},
                {"x": 8.25, "y": 3.25},
                {"x": 9.25, "y": 3.25},
                {"x": 10.25, "y": 3.25},
                {"x": 11.25, "y": 3.25},
                {"x": 12.25, "y": 3.25},
                {"x": 13.25, "y": 3.25},
                {"x": 14.25, "y": 3.25, "w": 2.25},
                {"x": 16.75, "y": 3.5},
                {"x": 0, "y": 4.25, "w": 1.5},
                {"x": 3, "y": 4.25, "w": 1.5},
                {"x": 4.5, "y": 4.25, "w": 2},
                {"x": 6.5, "y": 4.25, "w": 1.25},
                {"x": 8.25, "y": 4.25, "w": 2.75},
                {"x": 11, "y": 4.25, "w": 1.5},
                {"x": 15.75, "y": 4.5},
                {"x": 16.75, "y": 4.5},
                {"x": 17.75, "y": 4.5}
            ]
```

**Comment by noroadsleft** [comment]
As a general comment/FYI, the default keymap includes a `KC_END` key, but given the layout, I don't believe there is should be one, or at least not where it's currently said to be. I've ported the default keymap to QMK Configurator, and it looks, well, not right:

image

This would be a problem with the `maja.h` and any `keymap.c` files, rather than a problem here. But I'd consider that out of scope for this PR.

**Comment by moyi4681** [comment]
> As a general comment/FYI, the default keymap includes a `KC_END` key, but given the layout, I don't believe there is should be one, or at least not where it's currently said to be. I've ported the default keymap to QMK Configurator, and it looks, well, not right:
> 
> image
> 
> This would be a problem with the `maja.h` and any `keymap.c` files, rather than a problem here. But I'd consider that out of scope for this PR.

the keymap.c is not correct, I will fix it then

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**